cli/src/main.rs

```diff
@@ -320,7 +319,7 @@ async fn main() -> Result<()> {
     SERVAL_NODE_URL.lock().unwrap().replace(baseurl);
 
     match args.cmd {
```

Comment:
Should we just `.await?` this match statement rather than doing it in all of the arms?